uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/grpc/GRPCDiscoveryClient.java

```diff
@@ -125,6 +125,10 @@ public void onReceiveMqttMessage(int connectionId, MqttReceivedMessage message)
             msgBuilder.addAllProperties(message.getUserProperties());
         }
 
+        if (message.getContentType() != null && !message.getContentType().isEmpty()) {
```

Comment:
Please do not all getContentType() 3 times use variable 
`final String contentType = message.getContentType();`